backend/services/user/handler/user_test.go

```diff
@@ -0,0 +1,20 @@
+package handler_test
+
+import (
+	"testing"
+
+	pbcollab "github.com/kioku-project/kioku/services/collaboration/proto"
+	"github.com/kioku-project/kioku/services/user/handler"
+	"github.com/kioku-project/kioku/store"
+)
+
+func TestUser(t *testing.T) {
+	var mockUserStore store.UserStore = nil
+	var mockCollaborationService pbcollab.CollaborationService = nil
```

Comment:
`= nil` should be redundant

```suggestion
	var mockUserStore store.UserStore
	var mockCollaborationService pbcollab.CollaborationService
```

Reply:
Resolved in [8b4da82](https://github.com/kioku-project/kioku/pull/32/commits/8b4da820e50d5ad5da3413b4017c4aed1e68891f)